server/src/main/java/org/opensearch/cluster/action/index/MappingUpdatedAction.java

```diff
@@ -108,7 +108,7 @@ public void setClient(Client client) {
     /**
      * Update mappings on the cluster-manager node, waiting for the change to be committed,
      * but not for the mapping update to be applied on all nodes. The timeout specified by
-     * {@code timeout} is the cluster-manager node timeout ({@link ClusterManagerNodeRequest#masterNodeTimeout()}),
+     * {@code timeout} is the cluster-manager node timeout ({@link ClusterManagerNodeRequest#clusterManagerNodeTimeout()}),
      * potentially waiting for a cluster-manager node to be available.
      */
     public void updateMappingOnMaster(Index index, Mapping mappingUpdate, ActionListener<Void> listener) {
```

Comment:
Nitpick - should this method name be changed?

Reply:
Yeah, it should. I will rename and deprecate this method in issue #3544.
(Please see my explanation of the renaming plan in below https://github.com/opensearch-project/OpenSearch/pull/3617#issuecomment-1162444680)